org.gnome.gThumb.yml

```diff
@@ -0,0 +1,26 @@
+app-id: org.gnome.gThumb
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+
+command: gthumb
+rename-icon: gthumb
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
this needs --device=dri as well, it complains about lack of access to graphics drivers.

Reply:
interesting. it doesn't for me. But it makes sense to include this.